reaper/util.py

```diff
@@ -120,7 +120,7 @@ def write_state_file(path, state):
     os.rename(temp_path, path)
 
 
-def create_archive(content, arcname, metadata=None, outdir=None):
+def create_archive(content, arcname, rootdir=True, metadata=None, outdir=None):
```

Comment:
Good catch on this, as well. gephysio has been unmaintained for a while. There is more than one problem with that line. I'll add a general issue and point specifically to this line.